src/zelos/processes.py

```diff
@@ -492,7 +492,7 @@ def _load(self, p) -> None:
         self.logger.verbose(f"Loading process 0x{p.pid:x}")
         self.current_process = p
         if self.current_process.current_thread is None:
-            p.threads.swap_with_next_thread()
+            p.threads._swap_thread()
```

Comment:
This seems like a case where we would want to call the thread swap hook, since this changing the active thread within a process.